lib/reader.js

```diff
@@ -1,32 +1,29 @@
 'use strict';
 
 const stream = require('readable-stream');
-const isUndefined = require('lodash.isundefined');
 const JSONStream = require('JSONStream');
 const pack = require('browser-pack');
 const mergeStream = require('merge-stream');
-const depsSort = require('deps-sort');
-
+const through = require('through2');
 
 /**
  *
  * We need to manipulate order in all bundles to make sure that the
- * order is an increased value accross all bundles.
+ * order is an increased value across all bundles.
  *
  */
 
 class Order extends stream.Transform {
-    constructor () {
+    constructor (order) {
```

Comment:
Wont this get duplicates? The orderer starts at `arrayIndex`, but it orderes multiple packages, so the order increases, right? So the orderer might have assigned `1, 2, 3`, and then a new orderer that starts at `2` is created.